### ISSUE TITLE
Cross-compiling CUDA: next steps

### ISSUE THREAD

**Issue by h-vetinari** [conda-forge/nvcc-feedstock#95]
In https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/210, we tried to prepare the CI infra for CUDA cross-compilation. After sorting out some issues with the docker images (thanks @isuruf), we now hit (if using something like https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3624):
```
The reported errors are:
- Encountered problems while solving:
-   - nothing provides requested nvcc_linux-aarch64 11.2.*
```

It would be great if we could untangle the effort to enable cross-compilation (sorely needed for e.g. the arrow feedstock) from the separate [effort](https://github.com/conda-forge/staged-recipes/issues/21382) to bring CUDA >=12 into conda-forge.

CC @jakirkham

**Comment by h-vetinari** [comment]
Gentle ping @jakirkham

**Comment by h-vetinari** [comment]
Looks like this was solved by #96, thanks a lot @isuruf!
